Application Themes -  updated documentation (#671)
FusionAuth/fusionauth-site: site/docs/v1/tech/core-concepts/applications.adoc

+
This field is only displayed once multiple tenants exist in FusionAuth. When only a single tenant exists, the Application will always be created in the default tenant.

[field]#Theme# [optional]#Optional# [since]#Available since 1.27.0#::

> When a theme is selected this will be used instead of the theme configured by the tenant.

Reviewer: ```suggestion
When a theme is selected, it will be used for this application instead of the tenant theme. 
```

> 

Reviewer: ```suggestion

A paid edition of FusionAuth is required to use the application themes. 
 
```
